projects/batfish-client/src/test/java/org/batfish/client/ClientTest.java

```diff
@@ -894,6 +895,33 @@ public void testLoadQuestionFromTextInvalid() throws Exception {
   }
 
   @Test
+  public void testLoadQuestionsNames() throws Exception {
+    Client client =
+        new Client(new String[] {"-runmode", "gendatamodel", "-prettyanswers", "false"});
+    JSONObject testQuestion = new JSONObject();
+    testQuestion.put(
+        "instance",
+        new JSONObject()
+            .put("instanceName", "testQuestionName")
+            .put("description", "test question description"));
+    Path questionJsonPath = _folder.newFile("testquestion.json").toPath();
+    CommonUtil.writeFile(questionJsonPath, testQuestion.toString());
+    client._logger = new BatfishLogger("output", false);
+    client.processCommand(
+        new String[] {LOAD_QUESTIONS.commandName(), questionJsonPath.getParent().toString()}, null);
+    BatfishObjectMapper mapper = new BatfishObjectMapper();
+
+    // Reading the answer written by load-questions
+    Answer answerLoadQuestions =
+        mapper.readValue(client.getLogger().getHistory().toString(220), Answer.class);
```

Comment:
what is happening here? 220?

Reply:
I am getting the output level logs written by load-questions as that has the answer element, which I want to compare. I will change it to the constant from BfLogger